pkgs/applications/window-managers/sawfish/default.nix

```diff
@@ -0,0 +1,42 @@
+{ stdenv, fetchgit, pkgconfig, which, autoreconfHook, rep-gtk, pango, gdk_pixbuf, libXinerama, libXrandr, libXtst, imlib, gettext, texinfo, makeWrapper }:
+
+with stdenv.lib;
+stdenv.mkDerivation rec {
+
+  name = "sawfish-git-20150215";
```

Comment:
Hmm, anyway - you did wrong a bit, the date of commit is 31 Jan (not 15 Feb) as I can see in their repo.
Let's forgive it now, but hope this won't happen again.